profanities.go

```diff
@@ -3,68 +3,397 @@ package goaway
 var profanities = []string{
 	"anal",
 	"anus",
+	"arrse",
 	"arse",
-	"ass",
-	"ballsack",
+	"assfucker",
+	"asses",
+	"assfucker",
+	"assfukka",
+	"asshole",
+	"assholes",
+	"asswhole",
```

Comment:
```suggestion
	"ass",
```
You removed `ass` and replaced it with other things that would be handled by just `ass`

Reply:
I am sorry but as much as I'm trying to keep the straight face this review and comments are way too amusing 👍